projects/ngrx.io/content/guide/component/let.md

```diff
@@ -1,82 +1,89 @@
+The `*ngrxLet` directive is provided through the `ReactiveComponentModule`.
+To use it, add the `ReactiveComponentModule` to the `imports` of your NgModule:
 
-```typescript
+```ts
 import { NgModule } from '@angular/core';
 import { ReactiveComponentModule } from '@ngrx/component';
 
 @NgModule({
   imports: [
     // other imports
-    ReactiveComponentModule
-  ]
+    ReactiveComponentModule,
+  ],
 })
 export class MyFeatureModule {}
 ```
 
-## Comparison with Async Pipe
+## Comparison with `*ngIf` and `async`
 
-The current way of binding an observable to the view looks like that:
+The current way of binding an observable to the view looks like this:
 
 ```html
-<ng-container *ngIf="observableNumber$ | async as n">
-  <app-number [number]="n">
-  </app-number>
-  <app-number-special [number]="n">
-  </app-number-special>
+<ng-container *ngIf="number$ | async as n">
+  <app-number [number]="n"></app-number>
+  
+  <app-number-special [number]="n"></app-number-special>
 </ng-container>
  ```
 
-The problem is `*ngIf` is also interfering with rendering and in case of a falsy value the component would be hidden.
+The problem is that `*ngIf` is interfering with rendering.
+In case of `0` (falsy value), the component would be hidden.
 
-The `*ngrxLet` directive takes over several things while making it more convenient and safe to work with streams in the template.
+The `*ngrxLet` directive takes over several things and makes it more convenient
+and safe to work with streams in the template:
 
 ```html
-<ng-container *ngrxLet="observableNumber$ as n">
-  <app-number [number]="n">
-  </app-number>
+<ng-container *ngrxLet="number$ as n">
+  <app-number [number]="n"></app-number>
 </ng-container>
 
-<ng-container *ngrxLet="observableNumber$; let n">
-  <app-number [number]="n">
-  </app-number>
+<ng-container *ngrxLet="number$; let n">
+  <app-number [number]="n"></app-number>
 </ng-container>
 ```
 
-In addition to that it provides us information from the whole observable context.
-We can track the observable notifications:
+### Tracking Different Observable Events
 
-- next value
-- error value
-- completion state
+In addition to that it provides us information from the whole observable context.
+We can track next, error, and complete events:
 
 ```html
-<ng-container *ngrxLet="observableNumber$; let n; let e = $error, let c = $complete">
-  <app-number [number]="n"  *ngIf="!e && !c">
+<ng-container *ngrxLet="number$ as n; let e = $error; let c = $complete">
+  <app-number [number]="n" *ngIf="!e && !c">
   </app-number>
-  <ng-container *ngIf="e">
-  There is an error: {{e}}
-  </ng-container>
-  <ng-container *ngIf="c">
-  Observable completed: {{c}}
-  </ng-container>
+
+  <p *ngIf="e">There is an error.</p>
+  <p *ngIf="c">Observable is completed.</p>
 </ng-container>
 ```
 
+### Using Suspense Template
+
+There is an option to pass the suspense template that will be displayed
```

Comment:
I think it would be great to clarify what "suspense state" is.
Most of us know it from React, but for a new developer this would raise some questions without an answer (or the dev needs to look it up).